contrib/omrabbitmq/Readme

```diff
@@ -0,0 +1 @@
+Force PR to pass
```

Comment:
I guess this file needs to go away.

side-note:
```
$ git commit --amend
$ git push -f
```
also forces new full CI run.

Reply:
Change done